README.md

```diff
@@ -135,6 +128,25 @@ The [LogDNA Ingestion Key](https://app.logdna.com/manage/profile) associated wit
 
 The default app passed along with every log sent through this instance.
 
+##### Env
+
+* _**Optional**_
+* Type: `string`
+* Default: `''`
+* Values: `YourCustomEnvironment`
+* Max Length: `32`
+
+The default environment passed along with every log sent through this instance.
+
+##### FlushInterval
+
+* _**Optional**_
+* Type: `time.duration`
+* Default: `5`
+* Values: `10`
```

Comment:
How different are `Values` and `Default`?

Reply:
Am following convention here: https://github.com/logdna/python.

Default is what the value defaults to if not provided, and values are possible values that the user could use for this field

Reply:
oh, I see, @dm36, I'd say there is no need for `Values` - just leave `Default` if necessary

Reply:
I think default gives a view of what values are set to if not provided and values provides potential values that the user can set